backup.sh

```diff
@@ -9,7 +9,7 @@ export AWS_DEFAULT_REGION=us-east-1
 
 # Use trap to print the most recent error message & delete the restore instance
 # when the script exits
-function cleanup_on_exit {
+function cleanup_stale_instances {
 
   echo "Trap EXIT called..."
   echo "If this script exited prematurely, check stderr for the exit error message"
```

Comment:
You're going to output this error message even on a normal run .. that would be confusing.